Timezone - Scheduled Views (#5527)
SumoLogic/sumologic-documentation: blog-service/2025-06-27-manage.md

---
title: Timezone field for Scheduled Views (Manage)
image: https://help.sumologic.com/img/sumo-square.png
keywords:
  - timezone
  - manage
  - scheduled-view
hide_table_of_contents: true    
---

import useBaseUrl from '@docusaurus/useBaseUrl';

> We're happy to include the **Timezone** field for the Scheduled Views. Previously, *America/Los_Angeles (Includes DST)* was considered as the default timezone, going forward you can select the timezone of your choice. If you do not make a selection, the Scheduled View will default to the time zone from your browser.

Reviewer: from your browser , should this be said from your timezone preference in sumo ?

> We're happy to include the **Timezone** field for the Scheduled Views. Previously, *America/Los_Angeles (Includes DST)* was considered as the default timezone, going forward you can select the timezone of your choice. If you do not make a selection, the Scheduled View will default to the time zone from your browser.

Reviewer: Also, can we also inform, that user's can update the timezone of scheduled views as well, apart from selecting while creation

> We're happy to include the **Timezone** field for the Scheduled Views. Previously, *America/Los_Angeles (Includes DST)* was considered as the default timezone. Going forward, you can select/edit the timezone of your choice. If you do not make a selection, the Scheduled View will default to the timezone preference in Sumo Logic.

Reviewer 2: Should this be _will default to the timezone from your browser_?

Reviewer 3: If the user has set a timezone in the User Preferences page, that timezone will be used. otherwise, it will fall back to the browser's timezone.

> We're happy to include the **Timezone** field for the Scheduled Views. Previously, *America/Los_Angeles (Includes DST)* was considered as the default timezone. Going forward, you can select/edit the timezone of your choice. If you do not make a selection, the Scheduled View will default to the timezone preference in Sumo Logic.

Author: ```suggestion
We're happy to include the **Timezone** field for the Scheduled Views. Previously, *America/Los_Angeles (Includes DST)* was considered as the default timezone. Going forward, you can select/edit the timezone of your choice. If you do not make a selection, the Scheduled View will default to the timezone from your browser.
```
